<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UXLogViewer v1.1</string>
  </property>
  <property name="windowIcon">
   <iconset resource="UXLogViewer.qrc">
    <normaloff>:/Resources/LogView.svg</normaloff>:/Resources/LogView.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="verticalLayout_2"/>
  </widget>
  <widget class="QToolBar" name="m_pToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_pMenuBarFileOpen"/>
   <addaction name="m_pMenuBarFileSave"/>
   <addaction name="separator"/>
   <addaction name="m_pMenuBarFileChangeTheme"/>
   <addaction name="m_pMenuBarFileSyncTheme"/>
   <addaction name="m_pMenuBarFileSettings"/>
   <addaction name="separator"/>
   <addaction name="m_pMenuBarEditMessage"/>
   <addaction name="m_pMenuBarEditCopySelected"/>
   <addaction name="m_pMenuBarEditAddToResults"/>
   <addaction name="m_pMenuBarEditCopyResults"/>
   <addaction name="m_pMenuBarEditClear"/>
   <addaction name="separator"/>
   <addaction name="m_pMenuBarViewResultView"/>
   <addaction name="m_pMenuBarViewFilterAView"/>
   <addaction name="m_pMenuBarViewSubFilterAView"/>
   <addaction name="m_pMenuBarViewFilterBView"/>
   <addaction name="m_pMenuBarViewSubFilterBView"/>
   <addaction name="separator"/>
   <addaction name="m_pMenuBarHelpAbout"/>
  </widget>
  <widget class="QStatusBar" name="m_pStatusBar"/>
  <widget class="QMenuBar" name="m_pMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>624</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="m_pMenuBarFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="m_pMenuBarFileOpen"/>
    <addaction name="m_pMenuBarFileSave"/>
    <addaction name="action_Merge"/>
    <addaction name="separator"/>
    <addaction name="m_pMenuBarFileChangeTheme"/>
    <addaction name="m_pMenuBarFileSyncTheme"/>
    <addaction name="separator"/>
    <addaction name="m_pMenuBarFileSettings"/>
    <addaction name="separator"/>
    <addaction name="m_pMenuBarFileExit"/>
   </widget>
   <widget class="QMenu" name="m_pMenuBarEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="m_pMenuBarEditMessage"/>
    <addaction name="m_pMenuBarEditCopySelected"/>
    <addaction name="m_pMenuBarEditCopyResults"/>
    <addaction name="m_pMenuBarEditAddToResults"/>
    <addaction name="m_pMenuBarEditClear"/>
   </widget>
   <widget class="QMenu" name="m_pMenuBarView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="m_pMenuBarViewResultView"/>
    <addaction name="separator"/>
    <addaction name="m_pMenuBarViewFilterAView"/>
    <addaction name="m_pMenuBarViewSubFilterAView"/>
    <addaction name="separator"/>
    <addaction name="m_pMenuBarViewFilterBView"/>
    <addaction name="m_pMenuBarViewSubFilterBView"/>
   </widget>
   <widget class="QMenu" name="m_pMenuBarHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="m_pMenuBarHelpAbout"/>
   </widget>
   <addaction name="m_pMenuBarFile"/>
   <addaction name="m_pMenuBarEdit"/>
   <addaction name="m_pMenuBarView"/>
   <addaction name="m_pMenuBarHelp"/>
  </widget>
  <action name="m_pMenuBarFileOpen">
   <property name="icon">
    <iconset resource="UXLogViewer.qrc">
     <normaloff>:/Resources/IcoOpen.svg</normaloff>:/Resources/IcoOpen.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="m_pMenuBarFileSave">
   <property name="icon">
    <iconset>
     <normalon>:/Resources/IcoSave.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="m_pMenuBarFileSettings">
   <property name="icon">
    <iconset resource="UXLogViewer.qrc">
     <normaloff>:/Resources/IcoSettings.svg</normaloff>:/Resources/IcoSettings.svg</iconset>
   </property>
   <property name="text">
    <string>Se&amp;ttings</string>
   </property>
  </action>
  <action name="m_pMenuBarFileExit">
   <property name="icon">
    <iconset resource="UXLogViewer.qrc">
     <normaloff>:/Resources/IcoExit.svg</normaloff>:/Resources/IcoExit.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="m_pMenuBarEditCopySelected">
   <property name="icon">
    <iconset resource="UXLogViewer.qrc">
     <normaloff>:/Resources/IcoCopySelected.svg</normaloff>:/Resources/IcoCopySelected.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy Selected</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="m_pMenuBarViewFilterAView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="UXLogViewer.qrc">
     <normaloff>:/Resources/IcoFilterA.svg</normaloff>:/Resources/IcoFilterA.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Filter A View</string>
   </property>
  </action>
  <action name="m_pMenuBarViewResultView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="UXLogViewer.qrc">
     <normaloff>:/Resources/IcoResults.svg</normaloff>:/Resources/IcoResults.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Result View</string>
   </property>
  </action>
  <action name="m_pMenuBarViewSubFilterAView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="UXLogViewer.qrc">
     <normaloff>:/Resources/IcoFilterSubA.svg</normaloff>:/Resources/IcoFilterSubA.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Sub Filter A View</string>
   </property>
  </action>
  <action name="m_pMenuBarViewFilterBView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="UXLogViewer.qrc">
     <normaloff>:/Resources/IcoFilterB.svg</normaloff>:/Resources/IcoFilterB.svg</iconset>
   </property>
   <property name="text">
    <string>Filter &amp;B View</string>
   </property>
  </action>
  <action name="m_pMenuBarViewSubFilterBView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="UXLogViewer.qrc">
     <normaloff>:/Resources/IcoFilterSubB.svg</normaloff>:/Resources/IcoFilterSubB.svg</iconset>
   </property>
   <property name="text">
    <string>S&amp;ub Filter B View</string>
   </property>
  </action>
  <action name="m_pMenuBarEditAddToResults">
   <property name="icon">
    <iconset resource="UXLogViewer.qrc">
     <normaloff>:/Resources/IcoAddToResults.svg</normaloff>:/Resources/IcoAddToResults.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Add To Results</string>
   </property>
  </action>
  <action name="m_pMenuBarHelpAbout">
   <property name="icon">
    <iconset>
     <normalon>:/Resources/IcoAbout.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="m_pMenuBarFileChangeTheme">
   <property name="icon">
    <iconset resource="UXLogViewer.qrc">
     <normaloff>:/Resources/IcoThemeEdit.svg</normaloff>:/Resources/IcoThemeEdit.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Change Theme</string>
   </property>
  </action>
  <action name="m_pMenuBarEditCopyResults">
   <property name="icon">
    <iconset resource="UXLogViewer.qrc">
     <normaloff>:/Resources/IcoCopyResults.svg</normaloff>:/Resources/IcoCopyResults.svg</iconset>
   </property>
   <property name="text">
    <string>Copy Results</string>
   </property>
  </action>
  <action name="m_pMenuBarFileSyncTheme">
   <property name="icon">
    <iconset resource="UXLogViewer.qrc">
     <normaloff>:/Resources/IcoThemeSync.svg</normaloff>:/Resources/IcoThemeSync.svg</iconset>
   </property>
   <property name="text">
    <string>S&amp;ync Themes</string>
   </property>
  </action>
  <action name="m_pMenuBarEditClear">
   <property name="icon">
    <iconset resource="UXLogViewer.qrc">
     <normaloff>:/Resources/IcoClear.svg</normaloff>:/Resources/IcoClear.svg</iconset>
   </property>
   <property name="text">
    <string>C&amp;lear</string>
   </property>
  </action>
  <action name="m_pMenuBarEditMessage">
   <property name="icon">
    <iconset resource="UXLogViewer.qrc">
     <normaloff>:/Resources/IcoMessage.svg</normaloff>:/Resources/IcoMessage.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Show Message</string>
   </property>
  </action>
  <action name="action_Merge">
   <property name="text">
    <string>&amp;Merge</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="UXLogViewer.qrc"/>
 </resources>
 <connections/>
</ui>
